### DISCUSSION TITLE
I can not use text or rectangles on the output video

### DISCUSSION THREAD

**Discussion by soda117** [ultralytics/yolov5#7811] [Q&A]
Hello.
I have one question.
I put the following in the code of the detect.py and until last week I was able to write characters and rectangles in the output video.

#detect.py
line 171: save_one_box(xyxy, imc, file=save_dir / 'crops' / names[c] / f'{p.stem}.jpg', BGR=True)
#add
line172                n_1 = (det[:, -1] == 0).sum() 
line173                a = f"{n_1} "#{'A'}{'s' * (n_1 > 1)}, " 
line174                cv2.putText(im0, "counter " , (20, 50), 0, 1.0, (71, 99, 255), 3)
line175                cv2.putText(im0, "A : " + str(a), (20, 100), 0, 1.5, (71, 99, 255), 3)

However, when I entered the same code today, there were no characters or rectangles written on the output video.
I think the cause is that line 42 has been updated to dataloaders.
If you know of a solution, please let me know.
Thank you.

[detect_20220514.txt](https://github.com/ultralytics/yolov5/files/8692843/detect_20220514.txt)

**Comment by glenn-jocher** [comment]
@soda117 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by medic-lab** [comment]
> 

Where did you put that code on detect.py?
